### ISSUE TITLE
Need a "Notes" field

### ISSUE THREAD

**Issue by michellelocke** [AAFC-BICoE/dina-planning#281]
We need just a general field called "Notes".

For CNC DB, Notes consists of data that fits no where else. Sometimes we don't know what a piece of data means so we can't select (or create) an appropriate field. Additionally, we have data that we see occasionally, but it may not be worth creating additional fields to encompass these data. We also may not want to bloat out the database with excess fields, which will make data entry more complex, especially if managing a large group of students or contractors doing data entry.

I considered using Material Sample Remarks for this, but this field seems more like it is intended for remarks on the Material Sample Type and Material Sample State than for general notes about the Specimen (Sample) itself.

We also need the Notes field to be set up in such a way that it can encompass multiple Notes. We do not want a single text box where we have to separate notes with semicolons. We can have a lot of notes and if they are too long, data will get visually lost in long strings of text. It may also infer that unrelated pieces of info have a connection. See https://www.cnc.agr.gc.ca/taxonomy/Specimen.php?id=322505 as an example of a record with more than one Note, that are unrelated and need a place to be entered.

**Comment by OwenLonsdale** [comment]
Agreed. We have some specimens that need heavy note making. Some examples from some name-bearing types in the CNC I recently wrote a paper on: 

1) avikritanga Schmid 1968d, Moropsyche
holotype (1 male, pinned: minuten)
Verbatim: Inde (Manipur) Nungba 25-V-1960 F.Schmid / CNC 280210.
Note: Original species description incorrectly listed date of collection as the 22nd.

2) sakarawaka Schmid 1959b, Hydropsyche
holotype (1 male, pinned)
Verbatim: Iran (Ost.1) Majlar [sic] 17-V-1956 F.Schmid / CNC 236658.
Notes: No specimen with a determination or type label can be unambiguously attributed to this species, and only a single specimen in the CNC is curated in the unit tray for this species; while the holotype was collected on a published date of 18.v.1956, only a single specimen was listed in the original publication, making it most likely that this is the type specimen.

3) sukhabddha Schmid 1987, Leptocerus
holotype (1 male, pinned: minuten) (Figs 1, 2)
Verbatim: Inde (Kerala) Kalial 7-I-1962 F.Schmid / CNC 311695.
Notes: The specific epithet for this taxon is spelled quite inconsistently by Schmid; it is presented as ″sukhabadda″ on the type label, ″sukhabaddha″ on the CNC type number label and in the original work by Schmid (1987) in both the index (p. 146) and the species list (p. 138); the actual species description (Schmid, 1987: 117) presents the name as ″suk-habddha″, which is provisionally accepted here.

4) tarpaka Gordon & Schmid 1987 [In: Schmid (1987)], Setodes
holotype (1 male, ethyl alcohol)
Verbatim: THAILAND: chw. Chiang Mai, amp. Fang tambon Papah; Monungkate, April 19, 1975 A.E. Gordon, light trap / CNC 165695.
Notes: Original publication notes depository as TISTR, Thailand.

5) udayakara Schmid 1995a, Oecetis
holotype (1 male, pinned)
Verbatim: Inde (Madras) Kuttalam 15-I-1962 F.Schmid / CNC 311740.
Notes: The holotype labels for Oecetis prah-lada and O. udayakara were accidentally switched, as suggested by collection data on the label and in the description.

6) neboissi Schmid 1997, Psychomyia 
holotype[?] (1 male, pinned: minuten) 
Verbatim: Inde (C.) [=Cachar District] Bandarkhal 9-V-1960 F.Schmid / CNC 236358.
Notes: The description listed holotype data as “Manipour, Bongba Khulen, 28.VII.1960”. One dissected male paratype in the CNC was collected at this event, as were 6 other non-type specimens. The identity of the type should be considered tentative and requires verification. CNC Type No. 22253, but listed as “22254” in description.
